diff --git a/src/main/java/com/amazonservices/mws/client/MwsReader.java b/src/main/java/com/amazonservices/mws/client/MwsReader.java
index b9b5911..f283177 100644
--- a/src/main/java/com/amazonservices/mws/client/MwsReader.java
+++ b/src/main/java/com/amazonservices/mws/client/MwsReader.java
@@ -36,7 +36,7 @@
      * Returns null if the reader is not positioned on a labeled value
      * of the requested type.
      * <p>
-     * For XML reads: <code>&lt;name&gt;value&lt;/name&gt;
+     * For XML reads: <code>&lt;name&gt;value&lt;/name&gt;</code>
      * <p>
      * For JSON reads: <code>name:value</code> if in Object or
      * <code>value</code> if in list.
diff --git a/src/main/java/com/amazonservices/mws/client/MwsWriter.java b/src/main/java/com/amazonservices/mws/client/MwsWriter.java
index 6deece6..c2206a6 100644
--- a/src/main/java/com/amazonservices/mws/client/MwsWriter.java
+++ b/src/main/java/com/amazonservices/mws/client/MwsWriter.java
@@ -86,7 +86,7 @@
      * <p>
      * For JSON: <code>[,]label:value</code> or <code>[,]value</code>
      * <p>
-     * For XML: <code>&lt;label&gt;valueFragment&lt;/label&gt;
+     * For XML: <code>&lt;label&gt;valueFragment&lt;/label&gt;</code>
      * 
      * @param name
      *            The label for the value.
